blueprints/app/files/package.json

```diff
@@ -32,8 +32,8 @@
     "<% } %>@ember/optional-features": "^2.1.0",
     "@ember/string": "^3.1.1",
```

Comment:
v4 is non-breaking for new apps (breaking change was dropping support for super old ember)